cetmix_tower_webhook/__manifest__.py

```
@@ -0,0 +1,25 @@
+    "data": [
+        "security/ir.model.access.csv",
+        "views/cx_tower_webhook_authenticator_views.xml",
+        "views/cx_tower_webhook_log_views.xml",
+        "views/cx_tower_webhook_views.xml",
+        "views/res_config_settings_views.xml",
+        "views/menuitems.xml",
+    ],
+    "demo": [
+        "demo/demo_data.xml",
+    ],
+}
```

Pylint (3.3.7)

[warning] Statement seems to have no effect

(W0104)